extensions/replication/ReplicationConfigValidator.js

```diff
@@ -36,22 +51,22 @@ const joiSchema = joi.object({
             }).when('type', { is: 'role', then: joi.required() }),
         }).required(),
     },
-    destination: {
+    destination: joi.object({
         transport: transportJoi,
-        auth: joi.object({
-            type: joi.alternatives().try('account', 'role', 'service')
-                .required(),
-            account: joi.string()
-                .when('type', { is: 'account', then: joi.required() }),
-            vault: joi.object({
-                host: joi.string().optional(),
-                port: joi.number().greater(0).optional(),
-                adminPort: joi.number().greater(0).optional(),
-                adminCredentialsFile: joi.string().optional(),
-            }),
-        }).required(),
+        auth: destinationAuthJoi.optional(),
+        sites: joi.object().pattern(
+            joi.string(), // site name
```

Comment:
maybe we want to validate this is in the bootstrap list?

```suggestion
            joi.string().valid(Joi.in('bootstrapList')), // site name
```

Reply:
This won't work as the bootstrapList contains objects so we need a more complex custom validator, and i think `_validatePerSiteDestinationConfig` already does the job of validating what we need, no need to have an extra one just for this, sites that are not in the bootstrapList will just be ignored.